src/data/agile-quiz.ts

```diff
@@ -1,4 +1,56 @@
 const agileQuiz = [
+  {
+    Question:
+      "Which of the following is the primary role of a Scrum Product Owner?",
+    Answer: "Define and prioritize the product backlog",
+    Distractor1: "Lead daily standup meetings",
+    Distractor2: "Define and prioritize the product backlog",
```

Comment:
distractors are for wrong answers only

```suggestion
    Distractor2: "writes test for the product",
```